tests/unit/NumberUtilsTest.js

```diff
@@ -1,14 +1,13 @@
 const NumberUtils = require('../../src/libs/NumberUtils');
 
-const result = NumberUtils.rand64();
-
-const max64 = 9223372036854775807;
-
 describe('libs/NumberUtils', () => {
     it('should generate a random 64-bit numeric string', () => {
         const id = NumberUtils.rand64();
-        expect(isNaN(result)).toBe(false);
-        expect(result > max64).toBe(false);
+        expect(typeof id).toBe('string');
+        // eslint-disable-next-line no-undef
+        expect(BigInt(id)).toBeLessThanOrEqual(BigInt(9223372036854775807));
```

Comment:
Using BigInt here is safe because BigInt is defined in Node and we are using the node environment for jest?

Reply:
Can we not use string comparison?

Reply:
We could if we want to, this just _feels_ better IMO because it makes them numbers instead of JS black magic with strings

Reply:
Black magic? All programming languages (that we use at Expensify) do string comparisons 😕 🤷 NAB